inc/admin-bar.php

```diff
@@ -54,7 +58,10 @@ function setup_admin_bar(
 			'irving/admin-bar',
 			[
 				'config' => [
-					'iframe_src' => site_url( $path ),
+					'iframe_src' => add_query_arg(
+						$endpoint->custom_params,
+						site_url( $path )
+					),
```

Comment:
Needed to include the query params so that the admin bar accurately pulls in that context (ex. `?s=test`).